pkg/apis/kops/componentconfig.go

```diff
@@ -470,6 +470,8 @@ type KubeAPIServerConfig struct {
 	RequestheaderAllowedNames []string `json:"requestheaderAllowedNames,omitempty" flag:"requestheader-allowed-names"`
 	// FeatureGates is set of key=value pairs that describe feature gates for alpha/experimental features.
 	FeatureGates map[string]string `json:"featureGates,omitempty" flag:"feature-gates"`
+	// GoawayChance is the probability that send a GOAWAY to HTTP/2 clients. Default to 0, means never send GOAWAY. Max is 0.02 to prevent break the apiserver.
+	GoawayChance string `json:"goawayChance,omitempty" flag:"goaway-chance"`
```

Comment:
This seems to be defined as `float64`:
https://github.com/kubernetes/kubernetes/blob/b53b9fb5573323484af9a19cf3f5bfe80760abba/staging/src/k8s.io/apiserver/pkg/server/config.go#L266
```suggestion
	GoawayChance float64 `json:"goawayChance,omitempty" flag:"goaway-chance"`
```

Reply:
Correct, but controller-gen apparently does not like float64. When setting it, the output is 
```found float, the usage of which is highly discouraged, as support for them varies across languages. Please consider serializing your float as string instead. If you are really sure you want to use them, re-run with crd:allowDangerousTypes=true```

Please suggest if the flag crd:allowDangerousTypes=true can be added

Reply:
Thanks for the explanation. One more Q, how far would you like this flag to be backported?